Tella/Scenes/HomeView/Views/EditImageView/EditImageView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  EditImageView.swift
+//  Tella
+//
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+import Mantis
+
+struct EditImageView: View {
+    
+    @EnvironmentObject var sheetManager: SheetManager
```

Comment:
I think `sheetManager` is not used ?

Reply:
it is used to hide the previous sheetView after selecting the edit option from the bottomSheet
in line `24`

Reply:
maybe we need to hide it before presenting the edit image view ? what do you think ?

Reply:
I think we can keep it here, because otherwise we need to add hideView in several files before calling the EditImageView